layers/+completion/helm/funcs.el

```diff
@@ -540,16 +540,17 @@ Removes the automatic guessing of the initial value based on thing at point. "
 (defmacro spacemacs||set-helm-key (keys func)
   "Define a key bindings for FUNC using KEYS.
 Ensure that helm is required before calling FUNC."
-  (let ((func-name (intern (format "lazy-helm/%s" (symbol-name func)))))
+  (let* ((actual-func (if (consp func) (cdadr func) func))
```

Comment:
Since this macro expects unquoted function names in the normal case, I think it makes sense to also accept unquoted `("description" . func-name)` rather than with the quote.  Then this can just use `cdr` instead of `cdadr`---the latter is surprising.

Reply:
Done. BTW, we have a slight discrepancy in the quotation:
```
(spacemacs/set-leader-keys "<keys>"  '<func>)
(spacemacs/set-leader-keys "<keys>"  '("<desc>" . <func>))
```
vs.
```
(spacemacs||set-helm-key "<keys>"  <func>)
(spacemacs||set-helm-key "<keys>"  ("<desc>" . <func>))
```

It would be nice to have the same function signature for both `spacemacs||set-helm-key` and `spacemacs/set-leader-keys`.

Reply:
Yeah, agreed it would be more consistent.  I guess one reason it might be difficult is that `spacemacs||set-helm-key` needs its input to be a symbol and not just any old function (or perhaps a keymap, etc.).